web/docs/installation/install/manifest.md

```diff
@@ -1,6 +1,8 @@
-# Install
+---
+sidebar_label: Manually with manifests
```

Comment:
Can you put `Using kubectl` and `Using Glasskube` so there is no bias in the installation option?

Reply:
done